backend/config/env_vars.py

```diff
@@ -1,15 +1,27 @@
-from dotenv import load_dotenv
 from types import SimpleNamespace
 
+import json
 import os
 
-load_dotenv()
+# Definir la ruta del archivo de configuración
+config_path = os.path.expanduser('~/.config/lyra/config.json')
```

Comment:
It's better to make this without "~" using OS dependencies to load the home directory